packages/main/src/themes/TimePicker.css

```diff
@@ -0,0 +1,20 @@
+.ui5-timepicker{
+	display: inline-block;
```

Comment:
We have to support the html "hidden" attribute. If someone sets that to the ui5-timepicker, it should hide.

:host(:not([hidden])) {
	display: inline-block;
}

Reply:
Done